src/DataCollector/SentryAuthCollector.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Barryvdh\Debugbar\DataCollector;
+
+use DebugBar\DataCollector\DataCollector;
+use DebugBar\DataCollector\Renderable;
+use Illuminate\Support\Contracts\ArrayableInterface;
+use Illuminate\Foundation\Application;
+
+/**
+ * Collector for Laravel's Sentry Auth provider
+ */
+class SentryAuthCollector extends AuthCollector implements Renderable
+{
+    /**
+     * @var boolean
+     */
+    protected $showName = true;
+
+    public function __construct()
```

Comment:
Probably better to use dependancy injection and inject the real object, instead of using Facades?